Mathlib/RingTheory/RootsOfUnity/Basic.lean

```diff
@@ -664,7 +661,7 @@ def zmodEquivZPowers (h : IsPrimitiveRoot ζ k) : ZMod k ≃+ Additive (Subgroup
 @[simp]
 theorem zmodEquivZPowers_apply_coe_int (i : ℤ) :
     h.zmodEquivZPowers i = Additive.ofMul (⟨ζ ^ i, i, rfl⟩ : Subgroup.zpowers ζ) := by
-  rw [zmodEquivZPowers, AddEquiv.ofBijective_apply] -- Porting note: Original proof didn't have `rw`
```

Comment:
While I think it's reasonable to remove the earlier two porting notes here, I'm a bit less sure about this one. It seems to suggest that the old proof was just the below `exact`, and that something isn't equal in the way it was before, suggesting something may have gone wrong. Does the `exact` proof work?